README.md

```diff
@@ -1,14 +1,15 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2122S2-CS2103T-T17-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S2-CS2103T-T17-3/tp/actions)
 
 ![Ui](docs/images/Ui.png)
```

Comment:
Can consider fixing width and height to that of the original Tp image (726 × 593)